### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by johnbaldwin**
- Added a FAQ file
- Added a HISTORY file
- Added a PyPI specific README for the PyPI package description
- Updated the project setup.py to use the new PyPI specific README

You can see the PyPI specific README in the project description: https://test.pypi.org/project/figures/

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+RELEASE HISTORY
```
Comment (OmarIthawi) [inline HISTORY.md]: `HISTORY.md` -> `CHANGELOG.md`? Not important at all though.

File: setup.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
 import os
 from setuptools import find_packages, setup
 
-with open(os.path.join(os.path.dirname(__file__), 'README.rst')) as readme:
+with open(os.path.join(os.path.dirname(__file__), 'docs/readme-pypi.rst')) as readme:
```
Comment (OmarIthawi) [inline setup.py]: I can't find this file. Where am I supposed to look?

I did:
```
$ git checkout develop
$ git pull
$ git grep pypi
```

Nothing relevant showed up.

File: HISTORY.md (hunk 1 above)
Comment (johnbaldwin) [reply HISTORY.md]: I've been on this fence on this myself. I think I spent a couple of hours poring over how other repos did the changelog and faq docs. `CHANGELOG` *would* be easier to file search for in GitHub global search

File: setup.py (hunk 2 above)
Comment (johnbaldwin) [reply setup.py]: @OmarIthawi Dang! I forgot to add this file when I had to redo the branch. Added!

**Comment by johnbaldwin** [comment]
@OmarIthawi I'm also working up some more info in the main README to make it clear that it runs in Ginkgo but not yet other named releases. Also I'm adding instructions for submitting bugs

**Comment by johnbaldwin** [comment]
@OmarIthawi I renamed 'HISTORY' to 'CHANGELOG', added the missing PyPI readme and updated the main readme. The build failure is from the intermittent test bug, which I'm going to have to address soon as this is getting old

**Comment by johnbaldwin** [comment]
I just removed a docstring from one of the views. It wasn't helpful.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+RELEASE HISTORY
```
Comment (OmarIthawi) [inline CHANGELOG.md]:
```suggestion
CHANGELOG
```

Or, `Release History`

It just feels like someone's shouting.

Comment (johnbaldwin) [reply CHANGELOG.md]: Can do

Comment (johnbaldwin) [reply CHANGELOG.md]: Changed to 'Release History' since that is what it really is, w/out shouting

**Pull request merged**